### PR TITLE
Doc: DKIT add control profiling online guide

### PR DESCRIPTION
**Pull request by macDure**

### CODE REVIEW

File: docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+车辆控制评测是根据车辆自动驾驶的数据来进行控制评分评测，因此需要车辆完成车辆闭环自动驾驶，可以是循迹自动驾驶，也可以是带感知的自动驾驶。因此需要最少完成以下前提步骤：
+
+- [循迹搭建--车辆集成](Vehicle_Integration_cn.md)
+
+- [循迹搭建--定位模块配置](Localization_Configuration_cn.md)
+
+- [循迹搭建--Apollo系统安装](Apollo_Installation_cn.md)
+
+- [循迹搭建--车辆循迹演示](Start_Waypoint_Following_cn)
```
Comment (storypku) [inline docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md:37]: .md

File: docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+    | 模块 | channel名称 | 检查项目 |
+    |---|---|---|
+    | Canbus | `/apollo/canbus/chassis` | 确保能正常输出底盘数据 |
+    | Control | `/apollo/control` | 确保能正常输出控制信息数据，该数据需要启动自动驾驶后才能获得 |
+    | Planning | `/apollo/planning` | 确保能正常输出规划信息 |
+    | Localization | `/apollo/localization/pose` | 确保能正常输出位置数据 |
+    | GPS | `apollo/sensor/gnss/best_pose` | 确保`sol_type`为`NARROW_INT` |
+
+- 通过使用`cyber`命令来检查采集数据内是否存在上述`channel`的数据：
```
Comment (storypku) [inline docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md:75]: cyber_recorder

**Review by storypku — CHANGES_REQUESTED** [review]

**Comment by kevin-y-wang** [comment]
Great job! The only thing is that in the figure control_profiling_online_folder_structure.png, there is a typo in the title that "syetem"  should be "system". This is a historical typo. You may try to PS it or if easier, directly remove this title line from the figure.

File: docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md (hunk 1 above)
Comment (macDure) [reply docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md:37]: thank you

File: docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md (hunk 2 above)
Comment (macDure) [reply docs/specs/D-kit/Waypoint_Following/Control_profiling_Online_cn.md:75]: thanks

**Comment by macDure** [comment]
> Great job! The only thing is that in the figure control_profiling_online_folder_structure.png, there is a typo in the title that "syetem" should be "system". This is a historical typo. You may try to PS it or if easier, directly remove this title line from the figure.

Have fixed, thank you.

**Review by kevin-y-wang — APPROVED** [review]

**Review by storypku — APPROVED** [review]

**Pull request merged**